src/app/(client)/bakeries/page.tsx

```diff
@@ -1,14 +1,16 @@
-import React from "react";
+import React, { Suspense } from "react";
 import BakeryTab from "@/components/BakeryTab";
 import TitleSection from "@/components/TitleSection";
 
 export default function BakeryPage() {
   return (
     <section>
       <TitleSection title={"เบเกอรี่"} />
-      <div className="relative flex flex-col justify-center items-center">
-        <BakeryTab />
-      </div>
+      <Suspense>
```

Comment:
why use Suspense?

Reply:
useSearchParams needs to be wrapped in suspense